File: 03_Arreglos/Matrices/CSharp/ejemplo1.cs
using System;
class Programa {
    static void Main() {
        Console.Write("Filas: ");
        int f = int.Parse(Console.ReadLine() ?? "0");
        Console.Write("Columnas: ");
        int c = int.Parse(Console.ReadLine() ?? "0");

        int[,] A = new int[f, c];
        int[,] B = new int[f, c];
        int[,] C = new int[f, c];

        Console.WriteLine("\nIngrese elementos de A:");
        for (int i = 0; i < f; i++)
            for (int j = 0; j < c; j++) {
                Console.Write($"A[{i},{j}] = ");
                A[i, j] = int.Parse(Console.ReadLine() ?? "0");
            }

        Console.WriteLine("\nIngrese elementos de B:");
        for (int i = 0; i < f; i++)
            for (int j = 0; j < c; j++) {
                Console.Write($"B[{i},{j}] = ");
                B[i, j] = int.Parse(Console.ReadLine() ?? "0");
            }

        for (int i = 0; i < f; i++)
            for (int j = 0; j < c; j++)
                C[i, j] = A[i, j] + B[i, j];

        Console.WriteLine("\nMatriz C = A + B:");
        for (int i = 0; i < f; i++) {
            for (int j = 0; j < c; j++)
                Console.Write(C[i, j].ToString().PadLeft(6));
            Console.WriteLine();
        }
    }
}

File: 03_Arreglos/Matrices/Python/ejemplo1.py
def leer_matriz(f, c, nombre):
    print(f"\nIngrese elementos de {nombre}:")
    M = []
    for i in range(f):
        fila = []
        for j in range(c):
            fila.append(int(input(f"{nombre}[{i},{j}] = ")))
        M.append(fila)
    return M

def suma_matrices(A, B):
    f, c = len(A), len(A[0])
    return [[A[i][j] + B[i][j] for j in range(c)] for i in range(f)]

def imprimir_matriz(M, titulo):
    print(f"\n{titulo}:")
    for fila in M:
        print("".join(str(x).rjust(6) for x in fila))

def main():
    f = int(input("Filas: "))
    c = int(input("Columnas: "))

    A = leer_matriz(f, c, "A")
    B = leer_matriz(f, c, "B")
    C = suma_matrices(A, B)

    imprimir_matriz(C, "Matriz C = A + B")

if __name__ == "__main__":
    main()
